cli/src/main/kotlin/utils/PackageConfigurationOption.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.utils
+
+import org.ossreviewtoolkit.model.utils.PackageConfigurationProvider
+import org.ossreviewtoolkit.model.utils.SimplePackageConfigurationProvider
+
+internal sealed class PackageConfigurationOption {
+    data class File(val value: java.io.File) : PackageConfigurationOption()
```

Comment:
Nit: Maybe align the order of `Dir` vs. `File` with the one in `when` below`?

Also, using the FQN for `File` is a bit unusual here, esp. as you use it more than once. So how about importing it?

Reply:
First one done. Importing unfortunately doesn't work as it'd lead to a name clash.